web/src/PermissionListPage.js

```diff
@@ -110,11 +137,17 @@ class PermissionListPage extends BaseListPage {
 
     return (
       <Upload {...props}>
-        <Button type="primary" size="small">
+        <Button type="primary" size="small" ref={ref => this.ref3 = ref}>
```

Comment:
`ref={this.ref3` doesn't work? https://legacy.reactjs.org/docs/refs-and-the-dom.html